pkg/controllers/subnetset/subnetset_controller.go

```diff
@@ -69,12 +69,6 @@ func (r *SubnetSetReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 					updateFail(r, &ctx, obj, "")
 					return ResultRequeue, err
 				}
-				if obj.Spec.AccessMode == "" {
```

Comment:
I think we still need access mode in SubnetSet CRD

Reply:
updated.